osu.Game.Rulesets.Mania/UI/ColumnFlow.cs

```diff
@@ -101,6 +118,30 @@ public void SetContentForColumn(int column, TContent content)
             Content[column] = columns[column].Child = content;
         }
 
+        private void updateMobileSizing()
+        {
+            if (!IsLoaded || !RuntimeInfo.IsMobile || mobilePlayStyle.Value != ManiaMobilePlayStyle.ExtendedColumns)
```

Comment:
It seems like once a non-`ExtendedColumns` mode is chosen, it will never return to the expected state if returning to `ExtendedColumn`, unless I'm missing something.

Reply:
I didn't pay attention to the ability to change settings during gameplay, since it's usually hard to do so on mobile, but I can add handling if preferred.

Reply:
Yeah, it needs to work.